modules/grafana/templates/dashboards/publishing_api_overview.json.erb

```diff
@@ -829,6 +829,91 @@
         }
       ],
       "title": "New row"
+    },
+    {
+      "collapse": false,
+      "editable": true,
+      "height": "250px",
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "Graphite",
+          "editable": true,
+          "error": false,
+          "fill": 1,
+          "grid": {
+            "threshold1": null,
```

Comment:
Does the `@memorysize_mb` then come from the machine with the Grafana templates on, and not the machine running redis?